File: app/src/main/java/com/example/dcct/model/SignOutModel.java
package com.example.dcct.model;

import com.example.dcct.bean.BackResultData;

public interface SignOutModel {
    void postSignOutId(long uid,ReturnSignState returnSignState);
    interface ReturnSignState{
        void onComplete(BackResultData backResultData);
        void onError(String msg);
    }
}

File: app/src/main/java/com/example/dcct/model/SignOutModel.kt
package com.example.dcct.model

import com.example.dcct.bean.BackResultData

interface SignOutModel {
    fun postSignOutId(uid: Long, returnSignState: ReturnSignState)
    interface ReturnSignState {
        fun onComplete(backResultData: BackResultData<*>)
        fun onError(msg: String)
    }
}